custom/ilsgateway/zipline/helpers.py

```diff
@@ -0,0 +1,51 @@
+from pytz import timezone
+
+
+def format_date(datetime):
+    return timezone('Africa/Dar_es_Salaam').localize(datetime).strftime('%Y-%m-%d %H:%M:%S')
+
+
+def status_date_or_empty_string(status):
+    if status:
+        return format_date(status.timestamp)
+    else:
+        return ''
+
+
+def convert_products_dict_to_list(products_dict):
+    return [
+        '{} {}'.format(product_code, d['quantity'])
+        for product_code, d in products_dict.iteritems()
+    ]
+
+
+def delivery_lead_time(start_status, end_status):
+    if start_status and end_status:
+        return '%.2f' % ((end_status.timestamp - start_status.timestamp).seconds / 60.0)
```

Comment:
see the spec - sometimes we need to use timestamp, sometimes we need to use zipline_timestamp